### ISSUE TITLE
Installed ide-purescript and it's not working

### ISSUE THREAD

**Issue by safareli** [nwolverson/atom-ide-purescript#174]
Opening this as someone might have same issue:

```
$ apm remove ide-purescript
Uninstalling ide-purescript ✓
$ apm install ide-purescript@0.18.0
```

[reason](https://github.com/nwolverson/atom-ide-purescript/issues/172#issuecomment-331662581)

**Comment by nwolverson** [comment]
Sorry you got the wrong version - will fix this asap. But what wasn't working? Some basic things should still have worked...

**Comment by safareli** [comment]
Not sure, It was not working in a sense that in cmd+shift+p i had no `ide purescript` commands, didn't had  ` ✅ purescript` on the left bottom corner, no type annotations, I didn't knew why.

**Comment by nwolverson** [comment]
This should now be fixed, `0.18.0` is now also known as `0.19.1`...
